Project.toml

```
@@ -17,14 +17,13 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
 AbstractTrees = "0.3"
-DataFrames = "0.19.4, 0.20"
-DataStructures = "0.17.3, 0.17"
+DataFrames = "0.19.4, 0.20, 1.2"
```

Maybe

```suggestion
DataFrames = "0.19.4, 0.20, 1"
```

because `1.2` means "[1.2, 2.0)" according to the Pkg syntax https://pkgdocs.julialang.org/dev/compatibility/#Caret-specifiers